src/cljs/rems/dropdown.cljs

```diff
@@ -0,0 +1,53 @@
+(ns rems.dropdown
+  (:require [cljsjs.react-select]
+            [rems.guide-functions]
+            [rems.text :refer [text]])
+  (:require-macros [rems.guide-macros :refer [component-info example]]))
+
+;; TODO: add an ID to the input field and then link all labels to it
```

Comment:
Old comment?

Reply:
Thanks, I thought I deleted it but apparently not..